docs/_deployments/serverless-framework.md

```diff
@@ -0,0 +1,346 @@
+  // important because FaaS immediately terminate handlers after the response.
+  processBeforeResponse: true
+});
+
+// Initializes your app with your bot token and signing secret
+const app = new App({
+  token: process.env.SLACK_BOT_TOKEN,
+  receiver: expressReceiver
+});
+
+// Initialize your AWSServerlessExpress server using Bolt's ExpressReceiver
+const server = awsServerlessExpress.createServer(expressReceiver.app);
+```
+
+Finally, at the bottom of your app, update the [source code that starts the HTTP server](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L40-L45) to now respond to an AWS Lambda function event:
+
+```javascript
+// Handle the Lambda function event
+module.exports.handler = (event, context) => {
+  console.log('⚡️ Bolt app is running!');
+  awsServerlessExpress.proxy(server, event, context);
+};
+```
+
+When you're done, your app should look similar to the ⚡️[Getting Started with Serverless Framework app][getting-started-with-serverless-framework-app/app.js].
+
+**2. Add a serverless.yml**
+
+Every Serverless Framework app uses a special file called `serverless.yml` to configure and deploy your app.
+
+Create a new file called `serverless.yml` in your app's root directory and paste the following:
+
+```yaml
+service: serverless-bolt-js
+frameworkVersion: '2'
+provider:
+  name: aws
+  runtime: nodejs12.x
+  environment:
+    SLACK_SIGNING_SECRET: ${env:SLACK_SIGNING_SECRET}
+    SLACK_BOT_TOKEN: ${env:SLACK_BOT_TOKEN}
+functions:
+  slack:
+    handler: app.handler
+    events:
+      - http:
+          path: slack/events
+          method: post
+plugins:
+  - serverless-offline
+```
+
+> 💡 `SLACK_SIGNING_SECRET` and `SLACK_BOT_TOKEN` must be enviornment variables on your local machine.
+> You can [learn how to export Slack environment variables](/bolt-js/tutorial/getting-started#setting-up-your-local-project) in our Getting Started guide.
+
+**3. Install Serverless Offline** 
+
+To make local development a breeze, we can use the `serverless-offline` module to emulate a deployed function.
+
+Run the following command to install it as a development dependency:
+
+```bash
+npm install --save-dev serverless-offline
+```
+
+Congratulations, you've just prepared your Bolt app for AWS Lambda and Serverless! Now we can learn how to run and deploy the app.
+
+---
+
+### Run the app locally
+
+Let's learn how to run your Bolt app on your local machine, now that it's configured to respond to an AWS Lambda function.
+
+**1. Start your local servers**
+
+First, use the Serverless Framework to listen for an AWS Lambda function event:
+
+```zsh
+npx serverless offline --noPrependStageInUrl
+```
+
+Next, use ngrok to forward Slack events to your local machine:
+
+```zsh
+ngrok http 3000
+```
+
+> 💡 [Learn how to install ngrok](https://api.slack.com/tutorials/tunneling-with-ngrok) to create a public URL and forward requests to your local machine.
```

Comment:
Thanks! When I wrote this section, I debated referencing the install tutorial or Getting Started guide. I leaned on the tutorial because it was more focused, but I'll update it to the Getting Started guide because it adds more context!

Reply:
Updated in commit 7b93ff435ad830c747605bf022e8526b40b91ace to be:

> 💡 [Learn how to use ngrok][getting-started-guide-setting-up-events] to create a public URL and forward requests to your local machine